controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -218,15 +218,22 @@ def _mock_cli_object(cli_object):
         return Mock(as_single_element=cli_object)
 
     @staticmethod
-    def _get_cli_volume():
+    def _get_cli_volume(deduplicated_copy=True):
+        se_copy = YES
+        deduplicated_copy = 'no'
+        compressed_copy = 'no'
+        if deduplicated_copy:
+            se_copy = "no"
```

Comment:
please align the string quotation, at least here between the `no`s

Reply:
done